src/packages/cli/tests/args.test.ts

```diff
@@ -0,0 +1,56 @@
+import assert from "assert";
+import args from "../src/args";
+
+describe("args", () => {
+  describe("detach", () => {
+    const versionString = "Version string";
+    const isDocker = false;
+
+    const detachModeArgs = ["--detach", "--D", "--😈"];
```

Comment:
Should we add boolean as well here? Like `--detach=true`? Or do you think it's overkill?